render/animation/curve.go

```diff
@@ -47,6 +57,51 @@ func (cb CubicBezierCurve) Transform(t float64) float64 {
 	return math.NaN()
 }
 
+func (cb CubicBezierCurve) Reverse() Curve {
+	return CubicBezierCurve{1 - cb.c, 1 - cb.d, 1 - cb.a, 1 - cb.b}
+}
```

Comment:
Reversing a curve is useful when wanting to play an animation and then play it in reverse again.